libs/cookies/README.md

```diff
@@ -0,0 +1,79 @@
+# NgxCookies
+
+This library provides a quick and easy wrapper for [CookieConsent V3](https://cookieconsent.orestbida.com).
```

Comment:
Maybe, when this package has been released in a 1.0.0 version, we should reach out to them to see if there is a possibility of them also mentioning us on their page?

Reply:
Good idea :D